src/main/java/org/seattlevoluntech/controllers/UserController.java

```diff
@@ -50,5 +52,18 @@ public String fetchDataByLastName(@RequestParam("last_name") String lastName){
         return result.toString();
     }
 
+    @GetMapping("/user/listProjects/{userId}")
```

Comment:
Should be "/users/listProjects/{userId}"

Reply:
done